infra/aws/main.tf

```diff
@@ -20,6 +20,7 @@ module "aws_jumpbox" {
   vpc_subnet                = module.aws_base[0].vpc_subnets[0]
   cidr                      = module.aws_base[0].cidr
   tsb_version               = var.tsb_version
+  tsb_helm_repository       = var.tsb_helm_repository
```

Comment:
Can you please update the `DEVELOPMENT_BUILD.md` accordingly?

Reply:
ah you are right, good point!